examples/demo/cluster_grpc/main_quic.go

```diff
@@ -0,0 +1,160 @@
+package main
```

Comment:
Instead of creating a whole new main, why didn't you just add a flag to the existing main.go?

Reply:
The new main was created to avoid interfering with existing main code and maintain separation of responsibilities between execution modes. But we can easily adjust this.

Reply:
lets adjust it. I rather keep a single one